applications/nrf5340_audio/src/bluetooth/bt_renderer/bt_rend.h

```diff
@@ -42,14 +42,14 @@ int bt_rend_volume_set(uint8_t volume, bool from_vcp);
  *
  * @return	0 if success, error otherwise
  */
-int bt_rend_mute(bool from_vcp);
+int bt_rend_volume_mute(bool from_vcp);
```

Comment:
Use 'vol' here?

Reply:
No, the vol is in the file bt_rend_vol.h/c